src/mappings/stateChannel.ts

```diff
@@ -23,17 +24,22 @@ export async function handleChannelOpen(
   const {
     channelId,
     indexer,
-    consumer,
+    consumer: agent,
```

Comment:
May use `creator` to replace `agent` as this `consumer` can be the real consumer or c_h_contract

Reply:
Actually this `consumer` name should also changed in the contract event